Automattic/wp-calypso
# Stats: fix gated shortcuts

Related to https://github.com/Automattic/wp-calypso/pull/96995

## Proposed Changes

* The gating for shortcuts were broken when migrating it to date range picker. The PR proposes to fix them.

## Why are these changes being made?


* 

## Testing Instructions


* Create a simple sites and navigate to `https://wordpress.com/stats/day/:your-site`
* Open date range picker
* Ensure the shortcuts are gated except for 7 days
* Ensure when you click the gated shortcuts, it brings up the upsell overlay


<img width="776" alt="image" src="https://github.com/user-attachments/assets/997ed2c9-5da2-45f5-a42b-c62ef9994982">


<img width="829" alt="image" src="https://github.com/user-attachments/assets/112d560c-05cd-44e6-926b-8486cf75b3af">



## Pre-merge Checklist
